.gitignore

```diff
@@ -129,4 +129,7 @@ dist
 .yarn/install-state.gz
 .pnp.*
 
-appwrite.json
\ No newline at end of file
+appwrite.json
+
+# VSCode history directory
+.history
```

Comment:
I found 76 gitignore files with `.appwrite`:

![CleanShot 2024-07-09 at 17 19 26@2x](https://github.com/appwrite/templates/assets/19310830/413dff4c-296f-4afb-b6fe-7bfb5834b49c)


But I found 78 files with at least 1 character in `.gitignore`:

![CleanShot 2024-07-09 at 17 19 19@2x](https://github.com/appwrite/templates/assets/19310830/e46b93fb-7f29-4ce0-83b5-ed8735bb01ec)


I think that means we missed 2 gitignore files

Reply:
Nevermind. One is root level, and one is CI/CD action. You got them all